SC-workflow/Persist-Seq/Galaxy-Workflow-Persist-Seq-cell_cycle.ga

```diff
@@ -1,6 +1,7 @@
 {
     "a_galaxy_workflow": "true",
     "annotation": "Persist-Seq v0.4.0, Scanpy 1.9.3 (cell cycle)",
```

Comment:
Please make sure that the version here is updated as well. Possibly some very succinct indication of the change (ie. this being for Galaxy > x version) on the name or annotation,